admin_manual/enterprise_clients/publishing_ios_app.rst

```diff
@@ -0,0 +1,121 @@
+
+5. When it passes testing, upload (production) to the Apple store with Xcode
+
+Register on developer.apple.com
+-------------------------------
+
+See https://developer.apple.com/programs/ios/ for detailed information. It 
+costs $99 per year.
+
+Create provisioning profiles
+----------------------------
+
+1. Create a .certSigningRequest (CSR) file
+    * Open Keychain Access from Utilities    
+    * From Keychain Access toolbar select Keychain Access -> Preference
+    * Open Keychain Access > Certificate Assistant > Request a 
+      Certificate From a Certificate Authority
+    * Enter email address and common name that you used to register in the iOS 
+      Developer Program
+    * Keep CA Email blank and select "Saved to disk" and "Let me specify key 
+      pair information"
+    * Click Continue
+    * Choose a filename & destination on your hard drive
+    * Click Save
+    * In the next window, set "Key Size" value to "2048 bits"
+    * Set "Algorithm" to "RSA"
+    * Click Continue
+
+This will create and save your certSigningRequest file (CSR) to your hard drive. 
+A public and private key will also be created in Keychain Access with the Common 
+Name entered.
+
+2. Create ".cer" file in iOS developer account
+    * Login to https://developer.apple.com/
+    * Go to member center > certificates, identifiers, & profiles
+    * iOS Apps > Certificates > + button
+    * Production > App Store and Ad Hoc, Continue
+    * 'About Creating a Certificate Signing Request (CSR)' Continue
+    * Generate your certificate > Choose File > Generate
+    * new key is ios_distribution.cer. download and save
+    
+3. Create App IDs        
+    * Identifiers > App IDs, + to create new app ID
+    * Name, auto Team ID, bundle ID, Continue, confirm done
+    * Identifiers > App Groups, create app group, click done
+    * Identifiers > App IDs, edit app, add to App Group
+    * "You have successfully updated the App Groups associations with your App 
+      ID." 
+    * Identifiers > App IDs, create new DocumentProvider BundleID
+    * "This App ID is now registered to your account and can be used in your 
+      provisioning profiles."
+    * Edit DocumentProvider and add to App Group
+    * "You have successfully updated the App Groups associations with your App 
+      ID"
+    * Identifiers > App IDs, create new bundleID.DocumentProviderFileProvider. 
+    * Add to App Group
+
+4. Create Provisioning Profiles
+    * Provisioning Profiles > +, Ad Hoc, Select App ID from dropdown menu, 
+      generate, download
+    * Provisioning Profiles > +, Ad Hoc, Select the bundleID.DocumentProvider  
+      from dropdown menu, generate, download, click Add Another
+    * Provisioning Profiles > +, Ad Hoc, Select the  
+      bundleID.DocumentProviderFileProvider from dropdown menu, generate, 
+      download
+    
+5.  Register device UDIDs; you must register all devices that you will test 
+    your ad hoc app on   
+    
```

Comment:
I'd move this before creating the provisioning profile.
There is something to consider, once they have created the provisioning profile and attached the device UDIDs, if they add more UDIDs, they will have to edit the provisioning profiles, update them and send to us, otherwise, the ipa file won't work on the new devices